platform/CMakeLists.txt

```diff
@@ -1,2 +1,13 @@
+set(OPENSSL_USE_STATIC_LIBS TRUE)
+find_package(OpenSSL REQUIRED)
+
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
+
+find_library(LIB_RT rt REQUIRED)
```

Comment:
What are these for? If this is being introduced in this PR, can I assume it is only required by the OTA PAL? In that case would it make sense to move this to the OTA PAL specific CMakeLists.txt instead of this one?